ui/packages/app/web/src/components/Targets/utils.ts

```diff
@@ -0,0 +1,105 @@
+import moment from 'moment';
```

Comment:
I think we started converting things to date-fns, is there a specific functionality in moment that date-fns doesn't support?

Reply:
@brancz I'm all for using `date-fns`! just wanted to confirm this migration from moment is the case before making changes though.  Just asking because I still see several instances of 'moment' being used throughout the code.

Reply:
Whops, then that was definitely an oversight, I thought we had done the migration. Then let's use date-fns here and maybe open an issue for converting the other places where we use moment?

Reply:
turns out that I was able to avoid using an external date library (Moment or date-fns) altogether 😄